kumascript/macros/CSSRef.ejs

```diff
@@ -1,283 +1,582 @@
+        'CSS_layout_overview': 'CSS 版面配置概述',
+        'Normal_Flow': 'Normal Flow',
+        'Flexbox': '彈性區塊',
+        'Grids' : '格線',
+        'Floats': '浮動',
+        'Positioning': '定位',
+        'Multiple-column_Layout': 'Multiple-column Layout',
+        'Responsive_design': 'Responsive design',
+        'Media_queries': 'Beginner\'s guide to media queries',
+        'Legacy_Layout_Methods': 'Legacy Layout Methods',
+        'Supporting_Older_Browsers': 'Supporting Older Browsers',
+        'Fundamental_Layout_Comprehension': 'Fundamental Layout Comprehension',
+  },
+  'ja': {
+    'Learn_CSS': 'Learn CSS',
+      'CSS_first_steps': 'CSS first steps',
+        'CSS_first_steps_overview': 'CSS first steps overview',
+        'What_is_CSS':  'What is CSS?',
+        'Getting_started_with_CSS': 'Getting started with CSS',
+        'How_CSS_is_structured': 'How CSS is structured',
+        'How_CSS_works': 'How CSS works',
+        'Using_your_new_knowledge': 'Using your new knowledge',
+      'CSS_building_blocks': 'CSS building blocks',
+        'CSS_building_blocks_overview': 'CSS building blocks overview',
+        'Cascade_and_inheritance': 'Cascade and inheritance',
+        'CSS_selectors': 'CSS selectors',
+        'The_box_model': 'The box model',
+        'Backgrounds_and_borders': 'Backgrounds and borders',
+        'Handling_different_text_directions': 'Handling different text directions',
+        'Overflowing_content': 'Overflowing content',
+        'Values_and_units': 'Values and units',
+        'Sizing_items_in_CSS': 'Sizing items in CSS',
+        'Images_media_and_form_elements': 'Images, media, and form elements',
+        'Styling_tables': 'Styling tables',
+        'Debugging_CSS': 'Debugging CSS',
+        'Organizing_your_CSS': 'Organizing your CSS',
+      'Styling_text': 'Styling text',
+        'Styling_text_overview': 'Styling text overview',
+        'Fundamental_text_and_font_styling': 'Fundamental text and font styling',
+        'Styling_lists': 'Styling lists',
+        'Styling_links': 'Styling links',
+        'Web_fonts': 'Web fonts',
+        'Assessment_typesetting_a_community_school_homepage': 'Assessment: Typesetting a community school homepage',
+      'CSS_layout': 'CSS layout',
+        'CSS_layout_overview': 'CSS layout overview',
+        'Layout_introduction' : 'Introduction to CSS layout',
+        'Normal_Flow' : 'Normal Flow',
+        'Flexbox': 'Flexbox',
+        'Grids' : 'Grids',
+        'Floats': 'Floats',
+        'Positioning': 'Positioning',
+        'Multiple-column_Layout': 'Multiple-column Layout',
+        'Responsive_design': 'Responsive design',
+        'Media_queries': 'Beginner\'s guide to media queries',
+        'Legacy_Layout_Methods': 'Legacy Layout Methods',
+        'Supporting_Older_Browsers': 'Supporting Older Browsers',
+        'Fundamental_Layout_Comprehension': 'Fundamental Layout Comprehension',
+  }
+});
+
+
+const locale = env.locale;
+const slug = env.slug;
```

Comment:
I couldn't help but to notice that `slug` is never used. 
Meaning, every page that uses `CSSRef` gets the same HTML blog. 

I know about "premature optimization" and stuff but if we can know that the CSSRef output only depends on the `locale`, we could cache its output and re-use for every page that uses this macro! 
[Apparently this macro is the most time-coming of all](https://github.com/mdn/yari/issues/917). By a margin. So it would be a welcome addition to add a little bit of caching somewhere here.